tests/std/tests/Dev11_0000000_tuple_cat/test.cpp

```diff
@@ -169,6 +204,26 @@ int main() {
         assert(t == make_tuple(11, 22, 33U, s, 40, 50L, 60L, 'x', 'y', 'z', 1234LL));
     }
 
+    test_value_categories<pair>();
+    test_value_categories<tuple>();
+
+    {
+        array<int, 3> a1{{-1, -2, -3}};
+        const array<char, 4> a2{{'C', 'A', 'T', 'S'}};
+
+        {
+            auto cat5 = tuple_cat(a1, a2);

```

Comment:
No change requested, but `cat5` _really_ should be four pairs instead of two arrays.